Report/Template.Rmd

```diff
@@ -243,17 +251,267 @@ datasets = read.and.clean(dataset = "easySHARE_rel6_0_0.rda", wav = 99)
+
+Switzerland and the Nordic countries additionally present the least non-participation rates and the highest full-time working rates. 
+Summary Statistics show that those countries, which strive in health topics are related to high labor force participation rates in all categories. Despite this, the only candidate for multicollinearity is the age dummy in the age group 50, which is omitted before model calculation. 
+
+Generally, labor force participation is higher for males than for females. This applies to part-time work, too.
+
+The overall results of the descriptive statistics align well with @kalwij2005labour
+
+\newpage
+
 
 
 \section{Crosssectional probit regression}
-JULIAN
-\subsection{Theory and Design}
-\subsection{Implementation}
-\subsection{Empirical Results}
+
+## Theory and Design 
+
+Probit models allow for the estimation of binary, random response variables.  
+Probit and Logit models mitigate problems that would occur in case a (linear) regression would be used for a similar task. In this case, the range of the response variable would not be restricted to the interval (0,1). Consequently, as probabilities are commonly defined on this space, non-interpretable probability estimations would be delivered.
+The estimated coefficients are subject to a Wald Test on joint significance. 
+
+The model formula holds under the assumption that $\epsilon \sim N(0,1)$
+
+
+$$
+P(Y = 1 | X = x) = \Phi (x \beta)'
+$$
+
+Using this model, it is investigated how the probability of labor participation is influenced by health and age variables. 
+
+We are especially focussing on the calculation and evaluation of marginal effects.
+The marginal effect of $x_j$ on $P(Y = 1 | X = x)$ is defined as the effect of a ceteris paribus change of $x_j$ on the conditional probability that the response variable Y is realized as one.
+In case of a probit model, the marginal effect of $x_j$ on $P(Y = 1 | X = x)$ is dependent on x.
+
+
+## Implementation
+
+As stated in the 'Read and Clean' section, `df.splits` is obtained using the `split` function. A list is returned, that seperates the cleaned data frame by country and gender.
+This structure is helpful throughout the following sections, as it enables us to build a series of concatenated `lapply` functions. This function invokes a c-loop that iterates over each list element. As `lapply` takes a list as an input and returns a list, too; it preserves the initial structure. First, it is used to create a probit model for each subset within the list. One variable, namely the age 50 dummy variable, which is highly correlated with other regressors, is excluded due to multicollinearity. 
```

Comment:
It is perfectly correlated with the other age variables since we have an intercept!!!!